### ISSUE TITLE
Phosphaturic mesenchymal tumour

### ISSUE THREAD

**Issue by e-lbtn** [obophenotype/human-phenotype-ontology#6882]
_**Preferred term label:**_
phosphaturic mesenchymal tumour

**Definition (free text, please give PubMed ID)**
Cited in [PMID] : 28123260 ; 31301876 ; 33328210

**Parent term (use hpo.jax.org/app)**
Neoplasm by histology (HP:0011792)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Oncogenic osteomalacia (ORPHA:  352540)

**Comment by pnrobinson** [comment]
added as HP:0033987
